### ISSUE TITLE
[Dropdown][Accessibility]: Announces disabled field as clickable.

### ISSUE THREAD

**Issue by fsteffi** [OfficeDev/office-ui-fabric-react#11668]
Pre-requisite: Apply high contrast theme.

Environment Details:​​
Browser Versions: Chrome Version 78.0.3904.97
Screenreader: nvda2019.2
Windows Version: Windows10​​​

Repro Steps:​​
1. Hit the application URL
2. Repeat the same steps as ( Select application as "Sales hub" from the list of application--> From the change area which is present in the bottom select "App Settings"-->Navigate to "Site Map" and then Navigate to "Goal Management" and select "Forecast Configuration")
3. Fast Track Your Forecast Screen will appears.
4. Click on "Org chart forecast"
5. Forecast Configuration screen will appears
6. Fill all the mandatory field and click on "next button".
7.Launch screen reader as nvda

Actual:​
The combo-box under "Field" is announced as clickable, which is disabled.

Issue is not observed in narrator.

Same issue observed in the following page:
1.Configure.
2. General tab
3. Schedule tab
4. Fast your track grid -- for the column header its read clickable on which we cannot perform action.

Expected:​
The combo-box should be announced as "collapsed disabled"

User Impact:​
Screen reader users will receive the wrong information regarding the combo-box and try to perform actions on it.

[1639556_A11y_D365ForecastConfiguration_Forecastconfiguration_Permissions_ScreenReader.pptx](https://github.com/OfficeDev/office-ui-fabric-react/files/4044791/1639556_A11y_D365ForecastConfiguration_Forecastconfiguration_Permissions_ScreenReader.pptx)

**Comment by xugao** [comment]
@fsteffi - your repro details are your application-specific. in general, pls make your repro simple and minimal, and demo the bug outside of the context of your application code

**Comment by fsteffi** [comment]
@xugao
Reproduction of the issue in a codepen:
https://fabricweb.z5.web.core.windows.net/oufr/7.22.1/#/examples/dropdown
and
https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown

Steps to repro:
Hit the url provided navigate to the basic dropdown that is disabled and launch nvda screen reader and navigate to the disabled control using down arrow key

[dropdown.pptx](https://github.com/OfficeDev/office-ui-fabric-react/files/4052766/dropdown.pptx)

**Comment by xugao** [comment]
@fsteffi - Thanks for providing the repro details. Could you elaborate a bit on the steps how you navigated to the disabled control? The disabled control is designed to not be focusable

**Comment by fsteffi** [comment]
@xugao 
The steps to repro are:
Launch screen reader as nvda hit the url provided navigate to the basic dropdown section using down arrow key and navigate to the disabled combo-box "Disabled example with defaultSelectedKey" the "broccoli" is announced a s clickable while it is disabled.

**Comment by xugao** [comment]
@fsteffi - this seems an issue with NVDA, and it only repros in Chrome. I filed an issue with them for you: https://github.com/nvaccess/nvda/issues/10691. please follow up further there
